base/PerfOptions.php

```diff
@@ -107,6 +107,8 @@
   public ?string $scriptBeforeWarmup;
   public ?string $scriptAfterWarmup;
   public ?string $scriptAfterBenchmark;
+  public ?string $serverThreads = '100';
```

Comment:
Can this not be optionally null?  You set a default here, so it should never be null.